cmd/crates/soroban-rpc/Cargo.toml

```diff
@@ -1,50 +1,12 @@
 [package]
 name = "soroban-rpc"
-description = "Soroban RPC client for rust"
-homepage = "https://github.com/stellar/soroban-tools"
-repository = "https://github.com/stellar/soroban-tools"
+description = "Stellar RPC client for rust"
+homepage = "https://github.com/stellar/soroban-rpc"
+repository = "https://github.com/stellar/soroban-rpc"
 authors = ["Stellar Development Foundation <info@stellar.org>"]
 license = "Apache-2.0"
 readme = "README.md"
 version.workspace = true
```

Comment:
I have a second though on publishing a new soroban-rpc with a deprecated message. The problem is if we post a minor or patch version, as will happen here, folks might upgrade to it by accident. Also, it'd be odd to publish a v21. I think we could just drop publishing it or the moment. And then once we've moved over to stellar-rpc-client, we can yank all the soroban-rpc versions and that will display in the crates.io UI.

Reply:
Yeah it will be a breaking change when they upgrade bc they actually get empty crate. 

so will there be any problem with crates.io, If I rollback to initial option of removing` soroban-rpc` crate from the code and publishing new ` stellar-rpc-client `  @leighmcculloch

Reply:
No problem, I think that was the right approach and I sent us on an unnecessary detour, apologies for that.

Reply:
No worries, All good :)